fhir/resources/core/fhirabstractmodel.py

```diff
@@ -265,7 +269,13 @@ def parse_raw(
                 **extra,
             )
         except (ValueError, TypeError, UnicodeDecodeError) as e:  # noqa: B014
-            raise ValidationError([ErrorWrapper(e, loc=ROOT_KEY)], cls)
```

Comment:
This `ROOT_KEY` no longer exists. I don't yet know how it previously hooked into the pydantic internals